spec/requests/webhooks_controller_spec.rb

```diff
@@ -1,36 +1,67 @@
 # frozen_string_literal: true
 
 require "rails_helper"
-require_relative '../fabricators/webinar_fabricator.rb'
+require_relative "../fabricators/webinar_fabricator.rb"
 
 describe Zoom::WebhooksController do
-  let!(:zoom_id) { '1234' }
+  let!(:zoom_id) { "1234" }
+  let(:verification_payload) do
+    "{\"payload\":{\"account_id\":\"KlLYq9UvR3Cv5pcCVlQuNw\",\"object\":{\"uuid\":\"QOOZqdxqRVuyailQGyJHwA==\",\"participant\":{\"user_id\":\"16778240\",\"user_name\":\"Juan David Martínez Cubillos\",\"registrant_id\":\"E5qMfcA4SSmHgDMVSwYong\",\"participant_user_id\":\"E5qMfcA4SSmHgDMVSwYong\",\"id\":\"E5qMfcA4SSmHgDMVSwYong\",\"join_time\":\"2023-07-07T17:59:30Z\",\"email\":\"juan@discourse.org\",\"participant_uuid\":\"027419D6-2E3C-2861-BCCD-B4F67D41C260\"},\"id\":\"96766811781\",\"type\":5,\"topic\":\"Join test\",\"host_id\":\"E5qMfcA4SSmHgDMVSwYong\",\"duration\":300,\"start_time\":\"2023-07-07T17:59:30Z\",\"timezone\":\"America/Bogota\"}},\"event_ts\":1688752773282,\"event\":\"webinar.participant_joined\"}"
```

Comment:
There isn't a lot of json here, so this is fine. But to avoid weird formatting like this creating a json fixture is handy and then you can just load the json file without the \ formatting.